src/scss/components/estimation_least_squares/least-squares.scss

```diff
@@ -27,9 +27,9 @@
     padding-bottom: 1em;
 
     h2 {
-        font-size: 1.1em;
+        font-size: 1em;
         color: #000000;
-        margin-left: 2em
+        margin-left: 1.5em
```

Comment:
missing semicolon here? I'm not sure if scss even requires this, but the other lines here have semicolons.

Reply:
Good catch! Just pushed a fix.